yarn.lock

```diff
@@ -2300,6 +2300,20 @@
     prop-types "^15.7.2"
     react-is "^17.0.2"
 
+"@mui/base@5.0.0-alpha.77":
```

Comment:
Hmm, can you figure out why this is getting included here? It seems like this file shouldn't need to change at all if we just moved some deps into peer and dev deps, right?